core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -329,7 +331,7 @@ object Boot extends IOApp with LazyLogging {
         HealthMonitor.CheckAll
       )
 
-      val statusServiceConstructor: () => StatusService = () => StatusService.constructor(healthMonitor)
+      val statusServiceConstructor: () => StatusService = () => StatusService.constructor(healthMonitor)()
```

Comment:
What is the impact of this change? It seems like it would change the type.

Reply:
The parenthesis were implied before. The compiler was able to resolve it, but it's ambiguous and fragile.